Open Judge System/OJS.Common/Constants/CacheConstants.cs

```diff
@@ -19,9 +19,9 @@ public static class CacheConstants
 
         public const int DefaultPastContestsToTake = 15;
 
-        public static readonly string ContestResultsFormat = "ContestResults:{0}official:{1}full:{2}export:{3}";
+        public static readonly string ContestResultsFormat = "ContestResults:official:{0}:full:{1}:export:{2}:id:{3}";
```

Comment:
Results for export should not be cached.
Also It will be better to split cache for official and practice, as they might be cached for different timespans. Same applies for simple/full

Reply:
With the latest discussions the Results constants was removed and the results are not cached anymore